### DISCUSSION TITLE
Modifiche HTML Whistelblowing

### DISCUSSION THREAD

**Discussion by Vladj97** [globaleaks/globaleaks-whistleblowing-software#3945] [Q&A] [answered]
Ciao,

Vorrei chiedere se è possibile modificare il codice HTML del file home.html per personalizzare la vista dell'utente che intende fare una segnalazione. Sto osservando il progetto su GitHub, ma una volta installato localmente sembra che non siano disponibili tutte le cartelle. Come posso accedere al codice e apportare modifiche al Front-End?

Grazie anticipatamente.

**Comment by evilaliv3** [comment, marked as answer]
Grazie per la domanda.

Il software non prevede la possibilità di essere personalizzato a livello di HTML in quanto rischioso a livello di sicurezza.
Si prevede solo la possibilità di personalizzare tramite il cariamento di CSS e Javascript direttamente tramite l'interfaccia web di amministrazione.

**Reply by mpalian** [reply]
I also have the same issue. Is there a configuration guide to help us start trying this. All I have seen is the two simple examples in the User Documentation, but no context with which to connect it to the CSS in the repository. Any additional info would be valuable.
